content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -59,6 +59,8 @@ endpoints:
       topology.kubernetes.io/zone: us-west2-a
 ```
 
+Additionally, you must [enable Endpoint Slices](/docs/tasks/administer-cluster/enabling-endpoint-slices/#enabling-endpoint-slices) first.
```

Comment:
How about moving this detail to the start of the main contents (say line 26)?

Reply:
Also, the link to the task is on line 91 of this page.

Reply:
@kbhawkey, is it better to introduce this at the start?

Reply:
@sftim, moving this to the start is ok.

Reply:
@lojies , this page introduces the _alpha_ feature, Endpoint Slices, and does not necessarily provide steps to create a working example.

I don't think the page needs two links to the Endpoint Slices task page. If you want to add a statement about enabling the feature gate, you may want to add a bit more context.
Suggestions:

See the [What's next](/docs/concepts/services-networking/endpoint-slices#what-s-next) section for information about enabling Endpoint Slices.
or
You can follow the [Endpoint Slices task](/docs/tasks/administer-cluster/enabling-endpoint-slices/#enabling-endpoint-slices) to enable the Endpoint Slices feature.  

Page preview:
https://deploy-preview-17480--kubernetes-io-master-staging.netlify.com/docs/concepts/services-networking/endpoint-slices/

Reply:
@kbhawkey , thanks for the suggestion.I will close this pr.